### ISSUE TITLE
Passkey login request not intercepted by Bitwarden when using the "Use a passkey" context menu

### ISSUE THREAD

**Issue by sitiom** [bitwarden/clients#7456]
### Steps To Reproduce

1. Create a passkey for your GitHub account.
2. Log out of GitHub
3. On the login page, click the username field and press "Use a passkey"

### Expected Result

Bitwarden passkey popup prompt is launched.

### Actual Result

The default Windows Hello popup is launched instead.

### Screenshots or Videos

![image](https://github.com/bitwarden/clients/assets/56180050/8fdec1e8-71fe-4c8c-8915-c6d816282ec4)
Bitwarden popup not launched:
![image](https://github.com/bitwarden/clients/assets/56180050/c574ee72-0c79-4767-8c05-a52fe225ea1f)

### Additional Context

Do note that the "Sign in with a passkey" button works as usual.

Windows

### Web Browser

Brave

### Browser Version

Version 1.61.109 Chromium: 120.0.6099.144 (Official Build) (64-bit)

### Build Version

2023.12.1

### Issue Tracking Info

- [x] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by sitiom** [comment]
I do have the passkey stored in my account:
![image](https://github.com/bitwarden/clients/assets/56180050/7bb15422-a356-4747-9c79-d76327af1ba3)

**Comment by Krychaz** [comment]
Hello there,

I was indeed able to replicate this behaviour. It does look like the same issue as #7302 

Please share the details of your issue on that thread. I will close this issue in order to not make it too confusing for everyone.

**Comment by sitiom** [comment]
I was thinking it was a different issue as the latter issue was using a button over an autocomplete on a username field, but oh well.

**Comment by Krychaz** [comment]
That is true. However the common problem across these two issues is that while Passkey is saved within Bitwarden, the login process is not giving you an option to login using the saved Bitwarden Passkey.

**Comment by sitiom** [comment]
@Krychaz #7302 has been resolved, but the issue still isn't fixed for https://www.roblox.com/. Can you please reopen this issue?

**Comment by Havoc85** [comment]
Same in #6840 - still not fixed (in my example Nextcloud Login). Are there any news?

**Comment by kriswilk** [comment]
I'm having what seems to be the same problem on ebay.com. Reported in https://github.com/bitwarden/clients/issues/7785 just in case it turns out to be something else. But the symptoms are identical.

**Comment by RoFrie** [comment]
The same issue for me with ebay.de

**Comment by pamperer562580892423** [comment]
Like @RoFrie I have this issue also with ebay.de (I wrote details in [#7785](https://github.com/bitwarden/clients/issues/7785)).

And I can add, I just tried to create as well an "login-passkey" (for the vault) as an 2FA passkey for the vault, both for a free test Bitwarden account via the Bitwarden browser extension (there logged in with my premium account). In both situations, the browser extension didn't intercept the passkey-creation request - and vault.bitwarden.com/.eu ar not on my "excluded domains" list. Whether it makes sense to do all this is another question... but seems to be also the same kind of problem as I have with ebay.de.

**Comment by sitiom** [comment]
The autofill suggestions now show a list of passkeys that I can use, which works for me:
![Image](https://github.com/user-attachments/assets/4fcf820e-c7fa-4f43-924e-0e7c8bcaa5ab)
